lighthouse-core/report/html/report-styles.css

```diff
@@ -916,6 +884,47 @@ summary.lh-passed-audits-summary {
   object-fit: contain;
 }
 
+/* Chevron
+   https://codepen.io/paulirish/pen/LmzEmK
+ */
+
+.lh-chevron {
+  width: 12px;
+  height: 12px;
+  margin-top: calc((var(--body-line-height) - 12px) / 2);
+}
+
+.lh-chevron__lines {
+  transition: transform 0.4s;
+  transform: translateY(20px);
+}
+.lh-chevron__line {
+ stroke: hsl(216, 5%, 39%);
+ transform-origin: 50px 50px;
```

Comment:
can any of this be set to a variable to scale the chevron or is it all very precarious?

Reply:
good call. done.